diff -urN nautilus-actions-0.7.orig/nact/nact.desktop.in.in nautilus-actions-0.7/nact/nact.desktop.in.in
--- nautilus-actions-0.7.orig/nact/nact.desktop.in.in	2005-09-13 21:10:06.530781928 +0200
+++ nautilus-actions-0.7/nact/nact.desktop.in.in	2005-09-13 21:51:55.283393560 +0200
@@ -5,7 +5,5 @@
 Type=Application
 _Comment=Add items to the Nautilus context menu
 Exec=nautilus-actions-config
-Icon=@DATADIR@/pixmaps/nautilus-launch-icon.png
-Categories=Application;FileManager;DesktopSettings;Settings;AdvancedSettings;
-GenericName[en_US]=Nautilus Actions Configuration Tool
-Name[en_US]=Nautilus Actions Configuration
+Icon=@DATADIR@/nautilus-actions/nact/nautilus-launch-icon.png
+Categories=GTK;GNOME;Settings;AdvancedSettings;X-Capplet;
