diff -urN aaa/nautilus-actions-1.2/utils/Makefile.am nautilus-actions-1.2/utils/Makefile.am
--- aaa/nautilus-actions-1.2/utils/Makefile.am	2005-10-22 13:19:28.000000000 +0200
+++ nautilus-actions-1.2/utils/Makefile.am	2006-08-13 14:30:11.598852750 +0200
@@ -9,8 +9,8 @@
 	nautilus-actions-convert.c 				
 
 nautilus_actions_convert_LDADD =						\
-	$(NAUTILUS_ACTIONS_UTILS_LIBS)					\
-	$(top_builddir)/libnautilus-actions/libnautilus-actions.la
+	$(top_builddir)/libnautilus-actions/libnautilus-actions.la \
+	$(NAUTILUS_ACTIONS_UTILS_LIBS)
 
 nautilus_actions_new_config_SOURCES =	\
 	nautilus-actions-tools-utils.c	\
@@ -18,5 +18,5 @@
 	nautilus-actions-new-config.c 				
 
 nautilus_actions_new_config_LDADD =						\
-	$(NAUTILUS_ACTIONS_UTILS_LIBS)					\
-	$(top_builddir)/libnautilus-actions/libnautilus-actions.la
+	$(top_builddir)/libnautilus-actions/libnautilus-actions.la \
+	$(NAUTILUS_ACTIONS_UTILS_LIBS)
