--- nautilus-actions-3.0.7/src/plugin-menu/nautilus-actions.c~	2011-01-19 21:47:57.000000000 +0100
+++ nautilus-actions-3.0.7/src/plugin-menu/nautilus-actions.c	2011-04-21 20:46:37.140206073 +0200
@@ -86,7 +86,6 @@
 static void              menu_provider_iface_init( NautilusMenuProviderIface *iface );
 static GList            *menu_provider_get_background_items( NautilusMenuProvider *provider, GtkWidget *window, NautilusFileInfo *current_folder );
 static GList            *menu_provider_get_file_items( NautilusMenuProvider *provider, GtkWidget *window, GList *files );
-static GList            *menu_provider_get_toolbar_items( NautilusMenuProvider *provider, GtkWidget *window, NautilusFileInfo *current_folder );
 
 static GList            *get_menus_items( NautilusActions *plugin, guint target, GList *selection );
 static GList            *expand_tokens( GList *tree, NATokens *tokens );
@@ -395,7 +394,6 @@
 
 	iface->get_file_items = menu_provider_get_file_items;
 	iface->get_background_items = menu_provider_get_background_items;
-	iface->get_toolbar_items = menu_provider_get_toolbar_items;
 }
 
 /*
@@ -484,39 +482,6 @@
 			na_selected_info_free_list( selected );
 		}
 	}
-
-	return( nautilus_menus_list );
-}
-
-/*
- * as of 2.26, this function is only called for folders, but for the
- * desktop (x-nautilus-desktop:///) which seems to be only called by
- * get_background_items ; also, only actions (not menus) are displayed
- */
-static GList *
-menu_provider_get_toolbar_items( NautilusMenuProvider *provider, GtkWidget *window, NautilusFileInfo *current_folder )
-{
-	static const gchar *thisfn = "nautilus_actions_menu_provider_get_toolbar_items";
-	GList *nautilus_menus_list = NULL;
-	gchar *uri;
-	GList *selected;
-
-	g_return_val_if_fail( NAUTILUS_IS_ACTIONS( provider ), NULL );
-
-	if( !NAUTILUS_ACTIONS( provider )->private->dispose_has_run ){
-
-		selected = na_selected_info_get_list_from_item( current_folder );
-
-		if( selected ){
-			uri = nautilus_file_info_get_uri( current_folder );
-			g_debug( "%s: provider=%p, window=%p, current_folder=%p (%s)",
-					thisfn, ( void * ) provider, ( void * ) window, ( void * ) current_folder, uri );
-			g_free( uri );
-
-			nautilus_menus_list = get_menus_items( NAUTILUS_ACTIONS( provider ), ITEM_TARGET_TOOLBAR, selected );
-			na_selected_info_free_list( selected );
-		}
-	}
 
 	return( nautilus_menus_list );
 }
--- nautilus-actions-3.0.7/src/plugin-tracker/na-tracker.c~	2011-01-01 13:50:12.000000000 +0100
+++ nautilus-actions-3.0.7/src/plugin-tracker/na-tracker.c	2011-04-21 20:47:01.723541599 +0200
@@ -259,7 +259,6 @@
 
 	iface->get_background_items = menu_provider_get_background_items;
 	iface->get_file_items = menu_provider_get_file_items;
-	iface->get_toolbar_items = NULL;
 }
 
 static GList *
