diff -aurN nautilus-actions-0.99.orig/nact/nact.desktop.in nautilus-actions-0.99/nact/nact.desktop.in
--- nautilus-actions-0.99.orig/nact/nact.desktop.in	2005-10-28 15:32:52.000000000 +0200
+++ nautilus-actions-0.99/nact/nact.desktop.in	2005-11-24 21:23:53.000000000 +0100
@@ -6,4 +6,4 @@
 _Comment=Add items to the Nautilus popup menu
 Exec=nautilus-actions-config
 Icon=nautilus-actions-icon.png
-Categories=Application;FileManager;DesktopSettings;Settings;AdvancedSettings;
+Categories=GTK;GNOME;Settings;AdvancedSettings;X-Capplet;
